From d73adf56920fb2e45695992d2ec68cb603efd717 Mon Sep 17 00:00:00 2001
From: Vinay Verma <vvinay@codeaurora.org>
Date: Sun, 29 Apr 2018 14:08:30 +0530
Subject: [PATCH 04/14] power: Fix VNDK Compilation Errors

VNDK restrictions doesn't includes standard headers by
default, add them explicitly.

Change-Id: I4c72a28ffa141ecadcfe47cc7c1f62f2a0c8576e
---
 power-845.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/power-845.c b/power-845.c
index c7ddd04..9d9dd2c 100644
--- a/power-845.c
+++ b/power-845.c
@@ -38,6 +38,8 @@
 #include <fcntl.h>
 #include <dlfcn.h>
 #include <stdlib.h>
+#include <pthread.h>
+#include <unistd.h>
 
 #define LOG_TAG "QCOM PowerHAL"
 #include <log/log.h>
-- 
2.17.1

